evcc.dist.yaml

```diff
@@ -121,15 +121,17 @@ loadpoints:
     resetOnDisconnect: true # set defaults when vehicle disconnects
     soc:
       # polling defines usage of the vehicle APIs
+      # if CHARGING the polling interval is defined by the cache key on the vehicle definition (default 15m)
       # Modifying the default settings it NOT recommended. It MAY deplete your vehicle's battery
       # or lead to vehicle manufacturer banning you from API use. USE AT YOUR OWN RISK.
       poll:
-        # poll mode defines under which condition the vehicle API is called:
-        #   charging: update vehicle ONLY when charging (this is the recommended default)
-        #   connected: update vehicle when connected (not only charging), interval defines how often
-        #   always: always update vehicle regardless of connection state, interval defines how often (only supported for single vehicle)
+        # poll mode defines under which condition the vehicle API is called if NOT charging:
+        #   charging: update vehicle when connected and until charge process is done (w.r.t. targetSoc) (this is the recommended default)
```

Comment:
charging means charging and not connected. We're not updating when not charging. This seems wrong.

Reply:
I  know right?
But this is what was implemented in #4656.

Either we want to revert that PR now as "everybody" uses the SOC estimator.
Or we update the documentation to what the code actually does.
The current state seems to confuse a lot of people (see discussion).

Reply:
@Hofyyy könntest Du bitte mal den Kommentar lesen und in https://github.com/evcc-io/evcc/pull/4656/files#diff-c456526d4bd9022a488ce32de208b40d86016e2c40b0c81a2dffe557c186377dR1463 schauen? War es wirklich Absicht, hier "charging" zu "connected" zu ändern?

Reply:
Sorry so direkt zu sein, aber hat es so lange gedauert bis du gesehen hast, dass da "connected" uns nicht "charging" steht?

Denke schon, dass das so gedacht war.
In Zeile 1469 ( https://github.com/evcc-io/evcc/pull/4656/files#diff-c456526d4bd9022a488ce32de208b40d86016e2c40b0c81a2dffe557c186377dR1469) wird ja sowieso immer `True` zurückgegeben bei einem `lp.charging()` und der Interval ignoriert...

Reply:
> Sorry so direkt zu sein, aber hat es so lange gedauert bis du gesehen hast, dass da "connected" und nicht "charging" steht?

Du meinst seit August 22? Keine Ahnung was ich damals gesehen habe, jetzt ists mir jdfls aufgefallen. Oder meinst Du meine Reaktionszeit auf die Diskussion?

Ich werd nur leider aus Issue und PR nicht schlau. Bevor wir jetzt halbherzig die Doku ändern möchte ich erstmal verstehen, warum das Verhalten anders als erwartet ists.

Reply:
> Du meinst seit August 22? 

Ne eig seit [hier](https://github.com/evcc-io/evcc/issues/5890#issuecomment-1422533826) aber egal.

> Ich werd nur leider aus Issue und PR nicht schlau. Bevor wir jetzt halbherzig die Doku ändern möchte ich erstmal verstehen, warum das Verhalten anders als erwartet ists.

Das erscheint mir auch sinnvoller. Wenn man das Issue so durchließt scheint es so, dass das ein SOC estimator es verhindert hätte ([wie du auch geschrieben hast](https://github.com/evcc-io/evcc/issues/4220#issuecomment-1225824772)). Tendiere deshalb ehr zum Revert des PRs.

Reply:
Warten wir auf @Hofyyy. Ich glaube das war ein kosmetisches Problem: laden endet, zeigt aber nicht den target soc an. Der Fix tut jdfls mehr als ich erwartet hätte.

Reply:
Hi, ich bin die Tage auf Dienstreise. Meine damalige Änderung. Sollte forcieren das noch einmal der SOC geholt wird, wenn das laden beendet wurde. Da man z.b. bei 95% SOC stehen kann. Das lden dann beendet und nie wieder das auto gefragt wird. 

Der change war aber auch mehr ein workaround, da man beim beenden des laden kein reales update forcieren kann. Das bräuchte ein tieferes redesign. 

Wenn der Workaround stört, nehmt ihn raus.

Dann haz man halt im ui meistens nicht soc 100% stehen.

Reply:
Das connected wurde nur im zusammenhang mit soc < target soc eingebaut. es sollte also sehr deckungsgleich mit charging sein und nur den sonderfall am ende abdecken. JFI

Reply:
In meinen Augen ist das nicht wirklich deckungsgleich. Wenn ich das Auto abends anstecke, ist doch die ganze Nacht lang connected und soc < target soc.

Ich würde dann ehr das Redesign versuchen um nach Beendigung des Ladevorgangs trotz `mode: charging` noch ein einzelnen SOC Poll zu erlauben.

Reply:
Last das mal in #6267 weiter diskutieren.